src/mainFilesSidebar.js

```diff
@@ -75,18 +72,24 @@ const pinia = createPinia()
 
 store.dispatch('setMainContainerSelector', '.talkChatTab')
 
-const newCallView = () => new Vue({
-	store,
-	pinia,
-	render: h => h(FilesSidebarCallViewApp),
-})
+const newCallView = async () => {
+	const FilesSidebarCallViewApp = await import('./FilesSidebarCallViewApp.vue')
+	return new Vue({
+		store,
+		pinia,
+		render: h => h(FilesSidebarCallViewApp),
+	})
+}
 
-const newTab = () => new Vue({
-	store,
-	pinia,
-	id: 'talk-chat-tab',
-	render: h => h(FilesSidebarTabApp),
-})
+const newTab = async () => {
+	const FilesSidebarTabApp = await import('./FilesSidebarTabApp.vue')
```

Comment:
Probably, because it's a Promise, we can't access it
![image](https://github.com/nextcloud/spreed/assets/93392545/f7bae4bc-a5ef-4402-a8ae-dff8cf8465d9)

Also tab itself is missing:
![image](https://github.com/nextcloud/spreed/assets/93392545/0b1ebeee-663c-47a8-93da-f49895a7e92b)